aten/src/ATen/native/TensorAdvancedIndexing.h

```diff
@@ -14,7 +14,7 @@ namespace at { namespace native {
 
 enum class SCATTER_GATHER_OP: uint8_t {REDUCE_ADD, REDUCE_MULTIPLY, REDUCE_MAXIMUM, REDUCE_MINIMUM, REDUCE_MEAN};
 
-enum class INDEX_OP: uint8_t {PROD, MEAN, MINIMUM, MAXIMUM};
+enum class INDEX_OP: uint8_t {SUM, PROD, MEAN, MINIMUM, MAXIMUM};
```

Comment:
That's a pre-existing condition, but why do we have different enums for scatter_gather ops and for index ops, if conceptually they do the same thing - reduce elements that come to the same index.

Reply:
This enum was actually introduced by me, my bad. From discussion with Christian I think follow up work was to unify the code across index_reduce/scatter_reducce/segment_reduce (including this enum among other areas where code is duplicated) and clean it up but I can fix this in this PR instead!